site/src/api/typesGenerated.ts

```diff
@@ -2700,9 +2700,6 @@ export interface SessionLifetime {
 	readonly max_admin_token_lifetime?: number;
 }
 
-// From codersdk/client.go
-export const SessionTokenCookie = "coder_session_token";
-
```

Comment:
This isn't intentional; I need to figure out how to keep this around. 
@Emyrk think it's possible to modify this behaviour in `apitypings`?